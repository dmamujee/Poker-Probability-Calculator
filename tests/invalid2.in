6 H
12 S
13 S
3 C
11 H
12 S
12 S