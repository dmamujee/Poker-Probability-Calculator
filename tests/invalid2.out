
ERROR: game.cc: Given hand is same card as communal card
Card1: Queen of Spades
Card5: Queen of Spades
Hand Ranking: Invalid
